meta-xt-prod-devel-rcar-control/recipes-guests/doma/files/doma-set-root

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 # Change domain configuration to boot from storage configured by u-boot
 
+MODE="$1"
```

Comment:
It is not used anywhere

Reply:
Hi @lorc, please, read the commit message:

https://github.com/xen-troops/meta-xt-prod-devel-rcar/pull/103/commits/77df38522989452b1139aa93c1f7a0c3820fa194

```
To create conditions inside the doma(u)-set-root script an additional
'mode' parameter was added to it. As of now, it is used only for DomU,
but also added for DomA, just to have a consistent mechanism in place.
```

Reply:
Okay, but what is the point in introducing variable that is not used anywhere? This will confuse anyone reading code. If you really want to emphasize that this script could accept parameter, it is worth to add some comment akin "This script gets `MODE` parameter, which is not used right now"

Reply:
Hi @lorc, accepted.

Reply:
Hi @lorc, done.